tests/h/views/api/bulk/annotation_test.py

```diff
@@ -43,7 +43,9 @@ def test_it(
 
         bulk_annotation_service.annotation_search.assert_called_once_with(
             authority=pyramid_request.identity.auth_client.authority,
-            **valid_request["filter"],
+            audience=valid_request["filter"]["audience"],
```

Comment:
The test actually passed without this change but I reckon this makes for a better test.